README.md

```diff
@@ -36,6 +36,8 @@ A simple measure of dependency freshness
 
 `-r`, `--recursive`: search recursively for all compatible files, even if one is found in a directory passed as an argument
 
+`-o`, `--output`: sets how the output is displayed, valid options are `console` (default) or `json`. If `json` is selected, `--quiet` minifies the outputted json.
```

Comment:
Can we call it "table" and "json"? Since they're both outputting to the console...

Reply:
That makes entire sense!
I honestly have no idea why I did console...

Reply:
Let's capitalize JSON at the end of the sentence, too!